### PR TITLE
Split from groups actions PR 

### PR DESCRIPTION
**Pull request by KKoukiou**

### CODE REVIEW

**Review by martinpitt — COMMENTED** [review]
I think the "turn testgroup into a link" pixel changes need to be dropped from this PR, as that functionality isn't present yet. But otherwise LGTM!

File: pkg/users/logout-account-dialog.js [hunk 1]
```diff
@@ -35,7 +35,7 @@ export const logoutAccountDialog = (account) => {
                 style: "primary",
                 caption: _("Log out"),
                 clicked: () => {
-                    return cockpit.spawn(["/usr/bin/loginctl", "terminate-user", account.name], { superuser: "try", err: "message" })
+                    return cockpit.spawn(["loginctl", "terminate-user", account.name], { superuser: "try", err: "message" })
```
Comment (cockpituous) [inline pkg/users/logout-account-dialog.js:38]: This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18429-20230301-083127-e6f41828-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)

**Comment by martinpitt** [comment]
Yes, [what I predicted](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18429-20230301-083127-e6f41828-fedora-37/log.html#330). @KKoukiou please drop that one pixel ref change, then it should be green.

**Review by martinpitt — APPROVED** [review]
Epharisto!

File: pkg/users/logout-account-dialog.js (hunk 1 above)
Comment (cockpituous) [inline pkg/users/logout-account-dialog.js:38]: This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18429-20230301-103109-93b60a8e-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)

**Pull request merged**